dapr/proto/runtime/v1/dapr.proto

```diff
@@ -251,3 +255,16 @@ message UnregisterActorTimerRequest {
   string actor_id = 2;
   string name = 3;
 }
+
+// InvokeActorRequest is the message to call an actor.
+message InvokeActorRequest {
+  string actor_type = 1;
+  string actor_id = 2;
+  string method = 3;
+  bytes  data = 4;
+}
+
+// InvokeActorResponse is the method that returns an actor invocation response.
+message InvokeActorResponse {
+  google.protobuf.Any data = 1;

```

Comment:
Can we use bytes? This is more predictable behavior in different languages for SDKs.

Reply:
Changed.